include/secp256k1_frost.h

```diff
@@ -306,54 +306,60 @@ SECP256K1_API int secp256k1_frost_compute_pubshare(
     size_t n_participants
 ) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(2) SECP256K1_ARG_NONNULL(4) SECP256K1_ARG_NONNULL(5);
 
-/** Obtain the aggregate public key from a FROST x-only aggregate public key.
- *
- *  This is only useful if you need the non-xonly public key, in particular for
- *  ordinary (non-xonly) tweaking or batch-verifying multiple key aggregations
- *  (not implemented).
+/** Computes a group public key and uses it to initialize a keygen_cache
  *
  *  Returns: 0 if the arguments are invalid, 1 otherwise
- *  Args:        ctx: pointer to a context object
- *  Out:   ec_agg_pk: the FROST-aggregated public key.
- *  In: xonly_agg_pk: the aggregated x-only public key that is the output of
- *                    `secp256k1_frost_share_agg`
+ *  Args:         ctx: pointer to a context object
+ *  Out: keygen_cache: pointer to a frost_keygen_cache struct that is required
+ *                     for signing (or observing the signing session and
+ *                     verifying partial signatures).
+ *  In:     pubshares: input array of pointers to the public verification
+ *                     shares of the participants ordered by the IDs of the
+ *                     participants
+ *        n_pubshares: the total number of public verification shares
+ *              ids33: array of the 33-byte participant IDs of the signers
  */
-SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_frost_pubkey_get(
+SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_frost_pubkey_gen(
     const secp256k1_context *ctx,
```

Comment:
nit: `frost_pubkey_gen` -> `frost_keygen_cache_init`? 
The BIP calls it _KeygenCtxInit_

Reply:
I changed my mind. Libsecp APIs don't seem to use this `init` style. I'll have the BIP call it _KeygenCtx_ instead.